superset/assets/src/visualizations/deckgl/multi.jsx

```diff
@@ -33,7 +33,13 @@ function deckMulti(slice, payload, setControlValue) {
     // Filters applied to multi_deck are passed down to underlying charts
     // note that dashboard contextual information (filter_immune_slices and such) aren't
     // taken into consideration here
-    let filters = subslice.form_data.filters.concat(fd.filters);
+    let filters = [];
+    if (subslice.form_data.filters) {
+      filters = filters.concat(subslice.form_data.filters);
```

Comment:
Nit, can you use array spread here instead of `concat`?

```javascript
filters = [...filters, ...subslice.form_data.filters];
```

Same below, in the two uses of `concat`. It's the new preferred way of concatenating arrays.